Maestro/subscriptions.json

```diff
@@ -1351,11 +1351,11 @@
         }
       }
     },
-    // Update dependencies in WCF Master
+    // Update dependencies in WCF Main
     {
       "triggerPaths": [
         "https://github.com/dotnet/versions/blob/master/build-info/dotnet/standard/release/2.0.0/Latest.txt",
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/wcf/master/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/wcf/main/Latest.txt",
```

Comment:
It looks this a special case and we will also need to update vsoSourceBranch to main and /p:ProjectRepoBranch to main.

Reply:
This folder should be also renamed to main https://github.com/dotnet/versions/tree/master/build-info/dotnet/wcf/master. @mmitche please could you confirm?

Reply:
Rename of the folder  https://github.com/dotnet/versions/tree/master/build-info/dotnet/wcf/master isn't needed as it should be created automatically for main. I renamed master->main in action wcf-general.